examples/gen_primes.rs

```diff
@@ -7,42 +7,118 @@
 // of this source tree.
 
 use libpaillier::unknown_order::BigNumber;
-
+use openssl::{self, bn::BigNum, error::ErrorStack};
 use std::{
     fs::OpenOptions,
     io::prelude::*,
     time::{Duration, Instant},
 };
 
-/// Used to pre-generate 512-bit safe primes, so that they can
-/// be stored in a text file to make testing this library faster
+const PRIME_SIZE: usize = 1024;
```

Comment:
Ah, I did the prime generation before I made the change in the crate, so didn't try to re-use. It does seem more misuse-resistant to have a single value, but it would require making a bunch of things more public (e.g. the parameters module) and I'm not sure I want to mess with that without thinking carefully about our intended API.

Reply:
Ah okay. Yeah, we probably don't need to export that then